examples/Actor/README.md

```diff
@@ -33,14 +33,8 @@ The `ActorClient` project shows how to make client calls for actor using Remotin
 Run the client project from `ActorClient` directory as:
 
 ```sh
- dotnet run
- ```
-
- *Note: If you started the actor service with dapr port other than 3500, then set the environment variable DAPR_HTTP_PORT to the value of --dapr-http-port specified while starting the actor service before running the client in terminal.*
- ```
- On Windows: set DAPR_HTTP_PORT=<port>
- On Linux, MacOS: export DAPR_HTTP_PORT=<port>
- ```
+ dapr run --app-id actor_client dotnet run
```

Comment:
What's the reason for doing this? You don't need to run this app with a dapr sidecar to invoke the actors. I think showing that you can interact with actors from outside of dapr is an important point.

Reply:
Then both options should be added to the docs to explain the two ways.

In real scenarios, even if you are communicating with actors from the outside - this "outside" application will not run in the actors cluster - thus expecting dapr sidecar of the actors app is running on the same machine will not be possible.